config/locales/user_mailer/es.yml

```diff
@@ -15,6 +15,10 @@ es:
     password_changed:
       help: NOT TRANSLATED YET
       intro: NOT TRANSLATED YET
+    phone_changed:
+      help: NOT TRANSLATED YET
```

Comment:
I saw that, but noticed it wasn't exactly the same, so I didn't want to have an incorrect or inaccurate translation.